_drafts/oryx/setup.md

```diff
@@ -0,0 +1,37 @@
+# Setup
+
+You can install Oryx on your local environment. The installation typically takes a few minutes to be up and running. This guide explains how to set up your environment for Oryx framework development.
+
+## Prerequisites
+
+To install Oryx on your local system, you need the following:
+
+- [Node.js](https://nodejs.org/) or compatible Javascript runtime
+- npm package manager
+- A [Spryker storefront API](https://docs.spryker.com/docs/scos/dev/glue-api-guides/202204.0/glue-rest-api.html) (a public API is provided by default to get you up and running quick)
+
+## Installation
+
+To install Oryx you can use the [boilerplate project](https://github.com/spryker/composable-frontend) that we've prepared for you. In this repository you'll will find the minimum dependencies and configuration to install a standard Oryx application.
+
+The installation can be done with the following steps. This will install the dependencies and run the application.
+
+```
+git clone https://github.com/spryker/composable-frontend
+cd composable-frontend
+npm i
+npm run dev
+```
+
+Application will be available on `localhost:3000` by default.
+
+To better understand how the minimal boilerplate works, you can find more details in the [boilerplate guide](./boilerplate.md).
+
+## Builders/bundlers
+
+The best choice to start with Oryx framework is Vite, but Oryx is also compatible with every modern frontend builder/bundler, such as rollup, parcel, webpack, etc.
+The build tool must support [exports](https://nodejs.org/api/packages.html#package-entry-points) entry point in package.json as long as Oryx is using it in every package to split logical part of package into sub-packages.
```

Comment:
Wonder if we should talk about building here. I can imagine this deserves a dedicated section where we go into production builds.

Reply:
Agree that we should put this information somewhere, but as for me it's not relevant to setup. Maybe `packages` doc?

Reply:
new build doc I'd say

Reply:
Ok, created a task for it.